### ISSUE TITLE
Java version 11.0.15 from Microsoft is not supported as input value

### ISSUE THREAD

**Issue by AGulev** [actions/setup-java#329]
**Description:**
Microsoft has [11.0.15 version as LTS](https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11) version, but when I try to use it I get:
```
Error: Could not find satisfied version for SemVer 11.0.15. 
[31](https://github.com/defold/defold/runs/6577646803?check_suite_focus=true#step:4:32)
Available versions: 17.0.1, 16.0.2, 11.0.13
```

**Task version:**
Run actions/setup-java@v3
  with:
    java-version: 11.0.15
    distribution: microsoft

**Platform:**
- [x] Ubuntu
- [x] macOS
- [x] Windows

**Runner type:**
- [ ] Hosted
- [ ] Self-hosted

**Repro steps:**  
[A description with steps to reproduce the issue. If your have a public example or repo to share, please provide the link.](https://github.com/defold/defold/runs/6577646803?check_suite_focus=true)

**Expected behavior:**
11.0.15 works fine

**Actual behavior:**

>Error: Could not find satisfied version for SemVer 11.0.15.

**Comment by dmitry-shibanov** [comment]
Hello @AGulev. Thank you for your report. We'll investigate the issue. 

I think the issue is related to the fact that Microsoft Build of OpenJDK does not have a public API that is why the action uses predefined [versions](https://github.com/actions/setup-java/blob/main/src/distributions/microsoft/installer.ts#L79-L99). I consider we can add this version to the code.

**Comment by AGulev** [comment]
@marko-zivic-93 hi!
I see you have already merged my PR, I'm not sure how the release process looks like? It is available for using or it will be available only after the next version release? (3.4.0?)

**Comment by AGulev** [comment]
@e-korolevskii Thank you!
